defos/src/defos.cpp

```diff
@@ -7,7 +7,7 @@
 #endif
 #include <dmsdk/sdk.h>
 
-#if defined(DM_PLATFORM_OSX) || defined(DM_PLATFORM_WINDOWS) || defined(DM_PLATFORM_HTML5) || defined(DM_PLATFORM_LINUX)
+#if !defined(DM_HEADLESS) && (defined(DM_PLATFORM_OSX) || defined(DM_PLATFORM_WINDOWS) || defined(DM_PLATFORM_HTML5) || defined(DM_PLATFORM_LINUX))
```

Comment:
We added `DM_RELEASE`, `DM_DEBUG` and `DM_HEADLESS` recently.
Also, we removed more graphics related things/dependencies from our headless build.

Reply:
Since all `defos.*` functions were window/cursor related, it felt ok to simply disable the entire extension in headless mode.